code/src/java/pcgen/core/analysis/SizeUtilities.java

```diff
@@ -31,17 +33,14 @@ private SizeUtilities()
 	 * Get the default size adjustment
 	 * @return the default size adjustment
 	 */
-	public static SizeAdjustment getDefaultSizeAdjustment()
+	public static Optional<SizeAdjustment> getDefaultSizeAdjustment()
 	{
-		for (SizeAdjustment s : Globals.getContext().getReferenceContext()
-			.getConstructedCDOMObjects(SizeAdjustment.class))
-		{
-			if (s.getSafe(ObjectKey.IS_DEFAULT_SIZE))
-			{
-				return s;
-			}
-		}
+		return Globals.getContext()
+		              .getReferenceContext()
+		              .getConstructedCDOMObjects(SizeAdjustment.class)
+		              .stream()
+		              .filter(s -> s.getSafe(ObjectKey.IS_DEFAULT_SIZE))
+		              .findFirst();
```

Comment:
This should call findFirst() then get() since failure is not an option per my previous comment :)